metrics/resource.go

```diff
@@ -21,21 +21,24 @@ func getNodeResource(s stats.NodeStats, metadata *Metadata) *Resource {
 		LabelNodeName: s.NodeName,
 	}
 
+	resource := &Resource{
+		Type:   "node",
+		Name:   s.NodeName,
+		Labels: labels,
+	}
+	if s.CPU != nil {
+		resource.Timestamp = s.CPU.Time.Time
+	}
 	if metadata.IncludeNodeLabels {
```

Comment:
Can this `labels` creation go above the resource creation, where `labels` are used? I think it will work either way, but feels out of order now.

Reply:
I just added a new commit that moves the `labels` section above `Timestamp`.  I think the original creation of `resource` needs to happen before the `if` block, because `labels` includes `LabelNodeName: s.NodeName` at that point, but may not get more.